File: csharp/send_whatsapp_message.cs
using System;
using System.Net.Http;
using System.Text;
using System.Text.Json;
using System.Threading.Tasks;

class Program
{
    static async Task Main(string[] args)
    {
        string apiUrl = "<REPLACE_WITH_PROD_URL>/v1/whatsapp/send";
        string token = "<REPLACE_WITH_YOUR_API_KEY>";

        var httpClient = new HttpClient();
        httpClient.DefaultRequestHeaders.Add("Authorization", $"Bearer {token}");

        var requestData = new
        {
            from = "My Company",
            to = "2250000000000",
            templateName = "order_confirmation",
            parameters = new[] { "John", "12345" },
            mediaUrl = "https://mydomain.com/image.jpg",
            notifyUrl = "https://mydomain.com:4444/dlr"
        };

        var json = JsonSerializer.Serialize(requestData);
        var content = new StringContent(json, Encoding.UTF8, "application/json");

        var response = await httpClient.PostAsync(apiUrl, content);
        var responseBody = await response.Content.ReadAsStringAsync();

        Console.WriteLine(responseBody);
    }
}

File: java/send_whatsapp_message.java
import okhttp3.*;

import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        OkHttpClient client = new OkHttpClient().newBuilder().build();

        String prodUrl = "<REPLACE_WITH_PROD_URL>";
        String token = "<REPLACE_WITH_YOUR_API_KEY>";

        MediaType mediaType = MediaType.parse("application/json");
        String jsonBody = "{" +
                "\"from\": \"My Company\"," +
                "\"to\": \"2250000000000\"," +
                "\"templateName\": \"order_confirmation\"," +
                "\"parameters\": [\"John\", \"12345\"]," +
                "\"mediaUrl\": \"https://mydomain.com/image.jpg\"," +
                "\"notifyUrl\": \"https://mydomain.com:4444/dlr\"" +
                "}";

        RequestBody body = RequestBody.create(jsonBody, mediaType);

        Request request = new Request.Builder()
                .url(prodUrl + "/v1/whatsapp/send")
                .method("POST", body)
                .addHeader("Authorization", "Bearer " + token)
                .addHeader("Content-Type", "application/json")
                .build();

        try (Response response = client.newCall(request).execute()) {
            String responseBody = response.body().string();
            System.out.println(responseBody);
        }
    }
}
